tensorflow/lite/micro/examples/hello_world/zephyr_riscv/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(tf_lite_helloworld)
+
+set(TF_MICRO_DIR ${PROJECT_SOURCE_DIR}/../../../)
+file(GLOB MICRO_SOURCES ${TF_MICRO_DIR}/*cc)
```

Comment:
We've been trying to avoid duplicating the definitions of what files to build as our platform support expands, since this can be quite fragile as new files are added to the library. At the moment they only need to be added in one place in the makefile, but with this change they'd need to be added here as well.

The preferred approach is to generate a project file (in this case CMakeLists.txt) as a separate step, driven by the makefile build. You can see some examples of how we do that for Arduino, Mbed, etc here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/tools/make/helper_functions.inc#L115

Could you try rewriting this PR to use this method?

Reply:
@petewarden thanks for the review. We'll add a CMakeFiles.txt template and python script which will get the template, list of the source files and generate the final CMakeFiles.txt.

Is that OK?

Reply:
@petewarden I updated PR to generate CMakeLists.txt using `generate_project` from `helper_functions.inc` and it was not necessary to create extra python script. 

I do not copy third_party zephyr folder to generated `prj` folder  as it is large project. To compile generated project standalone, ZEPHYR_BASE environment variable should be set.

Could you review this PR again?